app/Policies/EventPolicy.php

```diff
@@ -9,6 +9,13 @@ class EventPolicy
 {
     use HandlesAuthorization;
 
+    public function before (User $user)
+    {
+        if($user->isAdmin()){
+            return true;
```

Comment:
`isAdmin` sam rzuca true/false - nie musisz robić ifa, wystarczy:
```suggestion
        return $user->isAdmin();
```

Reply:
https://laravel.com/docs/7.x/authorization#policy-filters

Reply:
Racja, nie wziąłem pod uwagę, że nie chcemy false rzucać. W takim razie popraw klamry